index.js

```diff
@@ -57,16 +39,6 @@ async function setup() {
   core.info(`Get local-cluster-dev with version `+ local_dev_cluster_version);
   shell.exec("git clone -b "+local_dev_cluster_version+" https://github.com/sustainable-computing-io/local-dev-cluster.git --depth=1");
   let parameterExport  = "";
-  if (cluster_provider === "kind"){
-    const kind_version = core.getInput('kind_version');
-    core.debug(kind_version);
-    if (kind_version !== undefined && kind_version!=null && kind_version.length!=0) {
-        core.info(`use kind version `+kind_version);
-      // kind_version, KIND_VERSION
-        parameterExport = parameterExport + "export KIND_VERSION="+kind_version;
-        parameterExport = parameterExport + " && "
-    }
-  }
   parameterExport = parameterExport + "export CLUSTER_PROVIDER="+cluster_provider;
```

Comment:
Do we need to have check  `if (parameterExport != "")`. We are already assigning it value here. In any case it will not be empty

Reply:
maybe not? we have a default runner https://github.com/sustainable-computing-io/local-dev-cluster/blob/main/main.sh#L141

Reply:
aha, I got your point.

Reply:
@vprashar2929 I just update this PR with a fix. :-) please help review.